### PR TITLE
import `workers.new` worker script

### PR DESCRIPTION
**Pull request by lukeed**
now w/ wrangler2 and typescript

### CODE REVIEW

**Comment by lukeed** [comment]
@eidam this doesnt deploy to staging environment, but i had the other tokens in place

File: packages/workers.new/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const [subdir, file] = redirects[pathname] || [];
+
+    if (subdir) {
+      const focus = encodeURIComponent(file);
+      const target = `https://stackblitz.com/fork/${source}/${subdir}?file=${focus}`;
+      return Response.redirect(target, 302);
+    }
+
+    // See the wiki below for a description of this URL works.
+    // https://wiki.cfops.it/display/ONB/How+to+use+Dashboard+Deep+Links
```
Comment (lukeed) [inline packages/workers.new/src/index.ts:28]:
```suggestion
```

File: .github/workflows/cd.workers-new.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    paths:
+      - 'packages/workers.new/**'
+
+jobs:
+  deploy:
+    name: Node.js v16
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: packages/create-cloudflare
```
Comment (eidam) [inline .github/workflows/cd.workers-new.yml:16]:
```suggestion
        working-directory: packages/workers.new
```

Comment (eidam) [reply .github/workflows/cd.workers-new.yml:16]: should this run in workers.new instead?

Comment (lukeed) [reply .github/workflows/cd.workers-new.yml:16]: yup thanks, i see the file is unsaved locally

**Review by eidam — APPROVED** [review]

**Pull request merged**